source/multigrid/mg_transfer_prebuilt.cc

```diff
@@ -284,23 +284,43 @@ void MGTransferPrebuilt<VECTOR>::build_matrices (
   // We keep track in the bitfield dof_touched which global dof has
   // been processed already (on the current level). This is the same as
   // the multigrid running in serial.
-  // Only entering on the finest level gives wrong results (why?)
+
+  struct dof_pair
```

Comment:
can you add a comment for this class? I think it would also be nice to name it `DoFPair` instead.